core/src/Microsoft.Identity.Core.csproj

```diff
@@ -1,8 +1,15 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras/1.5.4">
-
+  <Target Name="MacOSBuild" BeforeTargets="Build">
+    <Message Text="MAC_OS_BUILD = $(MAC_OS_BUILD)" Importance="High" />
+  </Target>
+  <PropertyGroup Condition="'$(MAC_OS_BUILD)' == ''">
+    <TargetFrameworks>net45;netstandard1.1;netstandard1.3;uap10.0;MonoAndroid8.1;Xamarin.iOS10;netcoreapp1.0</TargetFrameworks>
```

Comment:
>netcoreapp1.0< [](start = 95, length = 14)

mac should/could also include netcore ... possibly also android

Reply:
I will try compiling with these two targets

Reply:
you should be able to keep android as well. reason for this ask is to ensure that we can use the solution for as much as possible on a Mac (should someone prefer that as their dev setup)

---
In reply to: [235214338](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1418#discussion_r235214338) [](ancestors = 235214338)